smart_short_generator.py

```
@@ -418,78 +444,49 @@ def _validate_short_setup(self, setup: Dict[str, Any], daily_df: pd.DataFrame,
-            timeframe_aligned = self._check_timeframe_alignment(daily_df, weekly_df, setup['index'])
-            if self.short_thresholds['timeframe_alignment_required'] and not timeframe_aligned:
-                return None
-
-            if timeframe_aligned:
-                reasons.append("Multi-timeframe alignment confirmed")
+            # Layer 2: Basic Technical Check (RSI overbought only)
+            rsi_overbought = False
+            if 'RSI' in daily_df.columns and daily_df['RSI'].iloc[setup['index']] > 70:
+                reasons.append("RSI overbought")
                 confluence_score += 1
+                rsi_overbought = True
```

Ruff (0.12.2)

Local variable `rsi_overbought` is assigned to but never used

Remove assignment to unused variable `rsi_overbought`

(F841)